src/components/custom/_ctrader-wrapper.module.scss

```diff
@@ -0,0 +1,16 @@
+@use 'features/styles/theme/theme-mixins' as *;
+
```

Comment:
Hi @mohammad-hashemi-deriv can you please use css logical properties here 🙏 
Thanks man.